fdbserver/tester.actor.cpp

```diff
@@ -2730,6 +2730,10 @@ ACTOR Future<Void> runTests(Reference<AsyncVar<Optional<struct ClusterController
 	state bool enableDD = false;
 	state TesterConsistencyScanState consistencyScanState;
 
+	// Gives change for g_network->run() to let this run inside event loop and hence let
+	// the tests see correct value for `isOnMainThread()`.
```

Comment:
```suggestion
	// Gives chance for g_network->run() to run inside event loop and hence let
	// the tests see correct value for `isOnMainThread()`.
```

A bit more clear for me. Is it still conveying what you meant?